samples/web/content/apprtc/js/infobox.js

```diff
@@ -229,6 +234,30 @@ InfoBox.prototype.buildLine_ = function(label, value) {
   return line;
 };
 
+// Convert a number of milliseconds into a '[HH:]MM:SS' string.
+InfoBox.prototype.formatInterval_ = function(value) {
+  var result = '';
+  var seconds = Math.floor(value / 1000);
+  var minutes = Math.floor(seconds / 60);
+  var hours = Math.floor(minutes / 60);
+  var formatTwoDigit = function(twodigit) {
+    return ((twodigit < 10) ? '0' : '') + twodigit.toString();
+  };
+
+  if (hours > 0) {
+    result += formatTwoDigit(hours) + ':';
+  }
+  result += formatTwoDigit(minutes - hours * 60) + ':';
+  result += formatTwoDigit(seconds - minutes * 60);
+  return result;
+};
```

Comment:
This might be a good method to have a unit test for, depending if we intend the info box to be user accessible long term or not.